cmd/vflag/flag.go

```diff
@@ -0,0 +1,26 @@
+package vflag
+
+import (
+	"errors"
+	"strings"
+)
+
+// ValidatedFlag cli string type
+type ValidatedFlag struct {
+	Value string
+	Error string
+}
+
+func (f *ValidatedFlag) String() string {
+	return f.Value
+}
+
+// Set will be called for flag that is of validateFlag type
+func (f *ValidatedFlag) Set(value string) error {
+	if strings.Contains(value, "-") {
+		return errors.New(f.Error)
```

Comment:
I am not sure if this error message needs to be configurable since the check which verifies that it doesn't start with `-`  is always the same.

Reply:
I would just create an error here with the message `Flag value cannot start with -`